sdk/eventhub/azure-eventhubs/samples/sync_samples/client_secret_auth.py

```diff
@@ -23,5 +23,12 @@
                                   credential=credential)
 
 with producer:
-    event = EventData(body='A single message')
-    producer.send(event, partition_id='0')
+    event_data_batch = producer.create_batch(max_size=10000)
```

Comment:
Need to specify partition id

Reply:
when creating the batch partition_id can None, so I guess there is no need to add here?
Only if we want to show user how to send to a specific partition, which I had an example in the send.py